source/features/clean-header-search-field.tsx

```diff
@@ -0,0 +1,18 @@
+import onetime from 'onetime';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	(await elementReady<HTMLInputElement>('input.header-search-input'))!.value = '';
```

Comment:
Is the leading `input` for the type inference? If I'm right `element-ready` doesn't support it yet, and you are already specifying `HTMLInputElement` here.

```suggestion
	(await elementReady<HTMLInputElement>('.header-search-input'))!.value = '';
```